src/plone/api/group.py

```diff
@@ -17,7 +17,7 @@ def create(
     title=None,
     description=None,
     roles=[],
-    groups=[]
+    groups=[],
```

Comment:
we should keep this in mind, and either do this on this pull request or a new following one.

I think this repository did need a close look on conventions and best practices. I do see a lot of things that does look a bit scary.